.travis.yml

```diff
@@ -6,7 +6,13 @@ addons:
   postgresql: "9.4"
 services:
   - postgresql
-install: "pip install tox"
+install:
+  - pip install tox
+  - npm install jsonlint -g
 before_script:
   - psql -c 'create database filmfest;' -U postgres
-script: tox -e "py27-{sqlite,pg}"
+script:
+  - # --- Run JSON linter ---
+  - find . -name "*.json" -type f -exec jsonlint -q "{}" \; -exec /bin/echo "{}" \;
```

Comment:
in ideal world i'd like each json file to become a pytest test.

https://github.com/tholo/pytest-flake8/blob/master/pytest_flake8.py works this way.